packages/scripts/src/tasks/providers/ios.ts

```diff
@@ -45,6 +49,27 @@ const configure = async ({ appId, appName, versionName }: IiOSBuildOptions) => {
   });
 };
 
+const set_icons_and_splash_images = async (options: { assetPath: string }) => {
+  const { assetPath } = options;
+
+  const iconAndSplashSources = [];
+  if (fs.existsSync(assetPath)) {
+    iconAndSplashSources.push(assetPath);
+  } else {
+    return Logger.error({
+      msg1: "Icon and splash source assets not found",
+      msg2: `A source .png file is required to be used as a fall back for when the device's android version does not support adaptive icons. No asset was found at the path supplied in the deployment config: ${assetPath}.`,
+    });
+  }
+
+  const relativeAssetPath = path.relative(process.cwd(), assetPath).replace(/^(\.\.\/|\.\/)+/, "");
```

Comment:
See comment on line 68 below. I think this line could also use the `ROOT_DIR` value instead of `process.cwd()`, and avoid doing the regex replacement. Try logging out `ROOT_DIR` and `assetPath` at this point, and see if there's a method(s) of `path` that could be used to combine them and give the output you want

Reply:
Made the change here too.